src/index.ts

```diff
@@ -457,12 +457,16 @@ export class Bundler {
       typeof getFileName === 'function'
         ? getFileName({ format: rollupFormat, minify }, defaultFileName)
         : getFileName
-    const fileName = fileNameTemplate
+    let fileName = fileNameTemplate
       .replace(/\[min\]/, minPlaceholder)
       // The `[ext]` placeholder no longer makes sense
       // Since we only output to `.js` now
       // Probably remove it in the future
       .replace(/\[ext\]/, '.js')
+ 
+    if (rollupFormat === 'esm')  {
```

Comment:
```suggestion
    if (['es', 'esm', 'es-min', 'esm-min'].includes(rollupFormat))  {
```

Reply:
we want `es` format to continue using `.es` suffix to prevent from breaking change.